ansible/files/bootstrap/telemetry.sh

```diff
@@ -73,11 +45,103 @@ while [ $# -gt 0 ]; do
     shift
 done
 
+
+CURRENT_KEY=`cat $HOME/.ssh/authorized_keys`
+TELEMETRY_CONF_BUCKET=s3://{{telemetry_analysis_spark_emr_bucket}}
+MEMORY_OVERHEAD=7000  # Tuned for c3.4xlarge
```

Comment:
We have an issue for that out [0]. Would be interesting to do some experimentation on that point. Might be worth putting a bug out.

[0] https://github.com/mozilla/telemetry-analysis-service/issues/229